fuzzamoto-ir/src/generators/getaddr.rs

```diff
@@ -0,0 +1,39 @@
+use rand::RngCore;
+
+use crate::{
+    Operation,
+    generators::{Generator, GeneratorError, GeneratorResult, ProgramBuilder},
+};
+
+/// `GetAddrGenerator` emits a single `SendGetAddr` instruction targeting a random
+/// connection. Bitcoin Core disconnects peers that ask for addresses more than once (see
+/// `net_processing.cpp#L4679-L4713`), and the IR context doesn't yet expose connection direction,
```

Comment:
Probably best to avoid mentioning Bitcoin Core code directly because it might change and other implementations might not behave the same way.

The disconnections are fine, in fact we probably want to have the fuzzer violate the protocol sometimes to hit these cases.

The generators also generate into an existing program at a random point, so while you can check that there is no preceding `SendGetAddr` op you can't check if there is one later on in the program already.

https://github.com/dergoegge/fuzzamoto/blob/27c697998a84ac37544e0c5f9759c5fc48180ab9/fuzzamoto-libafl/src/mutators.rs#L175